### ISSUE TITLE
Orca screenreader getting stuck in the main content area and can't navigate by lines (Ubuntu, Linux)

### ISSUE THREAD

**Issue by cygnus-glacier** [edrlab/thorium-reader#2659]
I installed Thorium as an Appimage on Linux. The screenreader Orca does not work properly with this app, once jumping into the Main Content area it will try to read the Heading Level 1 and if there isn't one on screen because it is on another page when you try navigate somewhere else with Tab Orca will announce Tab repeatedly and give no context for where you are or where you're supposed to go. 

Even if there is text on screen you cannot use the shortcuts "P" to jump to the next paragraph, nothing will happen, though jumping to links or images with keyboard shortcuts seems to work for some reason. Pressing down to try go to the next line of text if you are at a heading will also just cause Orca to announce the key that was pressed but it will stay stuck in the heading.

### My Operating System:
```
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.5 LTS
Release:	22.04
Codename:	jammy

```

**Comment by gautierchomel** [comment]
I suspect this is due to #2519.

**Comment by danielweck** [comment]
Duplicate of https://github.com/edrlab/thorium-reader/issues/2519
